### DISCUSSION TITLE
How use usePopperTooltip with multiples elements rendered by map

### DISCUSSION THREAD

**Discussion by eliellima00** [mohsinulhaq/react-popper-tooltip#152] [Q&A] [answered]
I'am need to render multiples elements which  need to contain a tooltip, but how I can Make this using different refs and states provided by usePopperTooltip hook?

**Comment by denisborovikov** [comment, marked as answer]
You can create a `<Tooltip />` component and use multiple instances of this component wherever you need. See this example https://codesandbox.io/s/vigorous-hopper-mo5fpq?file=/src/index.js

**Comment by eliellima00** [comment]
thanks man

Em seg., 23 de mai. de 2022 às 15:21, Denis Borovikov <
***@***.***> escreveu:
